CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased][]
 
+## [0.10.3][] - 2024-01-03
```

Comment:
Don't you need to update the links at the bottom of the file?

Reply:
Derp, you're right, thanks!